troubleshooting/known_issues/kubernetes_node_rootFS_out_of_space.md

```diff
@@ -0,0 +1,26 @@
+# Kubernetes Master or Worker node's root filesystem is out of space
+
+## Description
+
+ There is a known bug in kubernetes 1.19.9 where movement of a pod with an attached volume may not complete in time and cause the kubelet service to stream error messages to the /var/log/messages log file.  If this goes unchecked it will fill up the root file system.
+
+## Fix
+
+1. Log into the node that has space issues.
+1. Verify that you have a large messages file in `/var/log/`.
+
+   ```bash
+   ncn-m/w:/var/log # ls -lh messages-20211212
+   -rw-r----- 1 root root 67G Dec 13 12:24 messages-20211212
+   ```
+
+1. Remove the file.
+1. Restart kubelet to address the streaming log entries.
+1. Restart the syslog service.
```

Comment:
The command to restart the syslog service should be put in case an operator looking at these instructions can't guess what the command is.

Reply:
updated